common-content/en/module/databases/databases-with-sheets/_index.md

```diff
@@ -0,0 +1,83 @@
+
+If you've used tools such as Excel or Google sheets, you will have seen this sort of structure before. We'll take a closer look at these in the backlog exercises to do this sprint.
+
+############ The below will be put into issues, just putting it in a PR to get any feedback :) #############
+
+# Backlog 1 - Learn about Google Sheets
+
+## Part 1 (Optional - Depending on knowledge of spreadsheets already.)
+
+Start by making sure we're familiar with Google sheets. The following tutorials will walk you through how to use a spreadsheet, format data, use functions, filters, and create charts using data.
+
+- [Build, organize, and format a spreadsheet using Google Sheets.](https://applieddigitalskills.withgoogle.com/c/college-and-continuing-education-uk/en-uk/google-workspace-sheets-part-1/google-workspace-sheets-part-1/important-lesson-update.html)
+
+- [Evaluate and interpret data with different spreadsheet functions, filters, and charts using Google Sheets.](https://applieddigitalskills.withgoogle.com/c/college-and-continuing-education-uk/en-uk/g-suite-certification-sheets-part-2/overview.html)
+
+- [Use advanced functions and conditional formatting in Google Sheets.](https://applieddigitalskills.withgoogle.com/c/college-and-continuing-education-uk/en-uk/g-suite-certification-sheets-part-3/overview.html)
+
+### Part 2 - Learn how to query data in Google Sheets using SQL
+
+Now you've got a good understanding of Google Sheets. We can start to use SQL, a powerful tool you will regularly come across when working with databases.
+
+What the video below, copying the dataset provided in the video and follow along with the queries as you go.
+
+- [SQL in Google Sheets](https://www.youtube.com/watch?v=dTNamL30sDg)
+- [Here is a link to the data](https://docs.google.com/spreadsheets/d/1zPeHxhKkYmdiqXNhMwhfC1A0o1wI0zNiD5Gsks2SYoc/template/preview)
+
+Do up to at the step `least GROUP BY clause and aggregation`. We can skip PIVOT, LIMIT and OFFSET for now, you wont need these for the next piece of coursework but feel free to look at them if you're curious and not short on time.
+
+# Backlog 2 - SQL Problems using Google Sheets
+
+Using the same data set we used in the Learn about Google Sheets coursework: [Here](https://docs.google.com/spreadsheets/d/1zPeHxhKkYmdiqXNhMwhfC1A0o1wI0zNiD5Gsks2SYoc/template/preview).
+We're going to practice extracting data from a database, using SQL.
+
+1. Emily Smith would like to be able to quickly see how many deals she's made. She only cares about `Deal Value` and `Expected Close date` to see when she'll be paid. Create a query that will give only this information.
+2. As the accountant working for this business, I need to work out the total sales we've made in each country to pay our taxes. Write a query to return a list of each closed deal (they have status of `"Closed Won"`) and sort them by country so we can work this out easily.
```

Comment:
```suggestion
2. As the accountant working for this business, I need to work out the total sales we've made in each country to pay our taxes. Write a query to return a list of each closed deal (they have status of `"Closed Won"`) and sort them by country so we can work this out easily.
```

Try to avoid nested dependent clauses. They cause comprehension problems for B2 speakers. [Hemingway](https://hemingwayapp.com/) can help you untangle pretty well. I generally aim for grade 4 but grade 6 is ok. Try for 0 "very hard to read" and as few as possible "hard to read"

Reply:
Ah yes ok! Manager to get this to grade 5